packages/twenty-server/src/business/modules/message/messaging.module.ts

```diff
@@ -0,0 +1,76 @@
+import { WorkspaceDataSourceModule } from 'src/engine/workspace-datasource/workspace-datasource.module';
+import { MessageParticipantModule } from 'src/business/modules/message/repositories/message-participant/message-participant.module';
+import { MessagingWorkspaceMemberListener } from 'src/business/modules/message/listeners/messaging-workspace-member.listener';
+import { MessagingMessageChannelListener } from 'src/business/modules/message/listeners/messaging-message-channel.listener';
+import { MessageService } from 'src/business/modules/message/repositories/message/message.service';
+import { WorkspaceMemberModule } from 'src/engine-workspace/repositories/workspace-member/workspace-member.module';
+import { FeatureFlagEntity } from 'src/engine/modules/feature-flag/feature-flag.entity';
+import { CreateCompaniesAndContactsModule } from 'src/engine-workspace/auto-companies-and-contacts-creation/create-company-and-contact/create-company-and-contact.module';
+import { CompanyModule } from 'src/business/modules/message/repositories/company/company.module';
+import { PersonModule } from 'src/engine-workspace/repositories/person/person.module';
+import { SaveMessagesAndCreateContactsService } from 'src/business/modules/message/services/save-messages-and-create-contacts.service';
+import { MessagingConnectedAccountListener } from 'src/business/modules/message/listeners/messaging-connected-account.listener';
+import { BlocklistModule } from 'src/business/modules/calendar-and-messaging/repositories/blocklist/blocklist.module';
+import { FetchByBatchesService } from 'src/business/modules/message/services/fetch-by-batch.service';
+@Module({
+  imports: [
+    EnvironmentModule,
+    WorkspaceDataSourceModule,
+    ConnectedAccountModule,
+    MessageChannelModule,
+    MessageChannelMessageAssociationModule,
+    MessageModule,
+    MessageThreadModule,
+    MessageParticipantModule,
+    CreateCompaniesAndContactsModule,
+    WorkspaceMemberModule,
+    TypeOrmModule.forFeature([FeatureFlagEntity], 'core'),
+    CompanyModule,
+    PersonModule,
+    BlocklistModule,
+    HttpModule.register({
+      baseURL: 'https://www.googleapis.com/batch/gmail/v1',
+    }),
+  ],
+  providers: [
+    GmailFullSyncService,
+    GmailPartialSyncService,
+    FetchMessagesByBatchesService,
+    GoogleAPIsRefreshAccessTokenService,
+    GmailClientProvider,
+    CreateContactService,
+    CreateCompanyService,
+    MessagingPersonListener,
+    MessagingWorkspaceMemberListener,
+    MessagingMessageChannelListener,
+    MessageService,
+    SaveMessagesAndCreateContactsService,
+    MessagingConnectedAccountListener,
+    FetchByBatchesService,
+  ],
+  exports: [
+    GmailPartialSyncService,
+    GmailFullSyncService,
+    GoogleAPIsRefreshAccessTokenService,
+    FetchByBatchesService,
+  ],
+})
+export class MessagingModule {}
```

Comment:
Not sure why this one was not a simple move. Also I think we want to keep the folder name as "messaging" and not "message" 🙂

Reply:
It was a simple move, but seems like git change his mind 😅
For the naming we should choose between, plural, singular, et conjugation like this one
I'm more for singular 🙂